src/Native/Runtime/assert.h

```diff
@@ -54,10 +54,21 @@ void Assert(const char * expr, const char * file, unsigned int line_num, const c
 
 #define FAIL_FAST_GENERATE_EXCEPTION_ADDRESS 0x1
 
-#define RhFailFast()  RhFailFast2(NULL, NULL)
+void RhFailFast();
```

Comment:
RhFailFast was a macro to get good file/line number from`ASSERT_UNCONDITIONALLY` used in `RhFailFast2`.